modules/simple_bucket/outputs.tf

```diff
@@ -33,3 +33,11 @@ output "internal_kms_configuration" {
   description = "The intenal KMS Resource."
   value       = var.internal_encryption_config.create_encryption_key ? module.encryption_key[0] : null
 }
+
+output "apphub_service_uri" {
+  value = {
+    service_uri = "//storage.googleapis.com/${google_storage_bucket.bucket.name}"
+    service_id  = substr(google_storage_bucket.bucket.name, 0, 63)
```

Comment:
Add location field similar to https://github.com/GoogleCloudPlatform/terraform-google-cloud-run/blob/main/modules/v2/outputs.tf#L86